src/lib/context.js

```diff
@@ -1,134 +1,97 @@
+import { Bundle } from './bundle';
 import { L10nError } from './errors';
-import { format } from './resolver';
-
-const IntlObjects = new WeakMap();
 
 export class Context {
-  constructor(env, langs, resIds) {
+  constructor(langs, resIds) {
```

Comment:
Not confusing for me. L20n stuff has many sources, Intl stuff only knows the locale code. :)